src/api/types.h

```diff
@@ -23,5 +28,74 @@ typedef struct CGameSettings {
   const char* seat_controllers[4];
   int num_controllers = 0;
 };
+
+typedef enum CMeldType {
+  kMeldChi = 0,
+  kMeldPon = 1,
+  kMeldKan = 2,
+  kMeldConcealedKan = 3,
+  kMeldNone = 4 // Default value
```

Comment:
I would make kMeldNone -1 instead of 4. I think that's more understandable as the enum value here.

Reply:
oh for some reason I thought these were unsigned, great!